packages/tao/src/shared/detect-package-manager.ts

```diff
@@ -21,28 +12,29 @@ function isPackageManagerInstalled(packageManager: string) {
   }
 }
 
-export async function detectPackageManager(
-  host: virtualFs.Host<Stats>
-): Promise<string> {
-  const hostTree = new HostTree(host);
-  if (hostTree.get('workspace.json')) {
-    const workspaceJson: { cli: { packageManager: string } } = JSON.parse(
-      hostTree.read('workspace.json').toString()
+export function detectPackageManager() {
+  const workspaceJsonPath = [`workspace.json`, `angular.json`].find((p) =>
+    existsSync(p)
+  );
+
+  if (workspaceJsonPath) {
+    const workspaceJson = JSON.parse(
```

Comment:
Idk why this wasn't using it before but we have a function called `readJsonFile` that also handles comments in JSONs.

Reply:
`readJsonFile` is part of the `@nrwl/workspace` and couldn't import it here 😃 do you want me to strip comments also?